pkg/plugin/plugin.go

```diff
@@ -170,7 +170,8 @@ func (p *Plugin) CmdAdd(args *skel.CmdArgs) error {
 		}
 
 		if len(newResult.IPs) == 0 {
-			return errors.New("IPAM plugin returned missing IP config")
+			err = errors.New("IPAM plugin returned missing IP config")
+			return err
```

Comment:
why this change?

Reply:
This required to trigger cleanup logic, e.g. here
https://github.com/ykulazhenkov/accelerated-bridge-cni/blob/c7644272fc12e3756e3a55fac54b9fb0d31f8356/pkg/plugin/plugin.go#L161